### PR TITLE
Add label support in choice questions for webapps

### PR DESCRIPTION
**Pull request by biyeun**
https://dimagi-dev.atlassian.net/browse/SAASP-10091

##### SUMMARY
Add support for `label` and `list-nolabel` appearance attributes to choice questions in webapps.

### CODE REVIEW

**Review by emord — APPROVED** [review]

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js [hunk 1]
```diff
@@ -331,6 +331,41 @@ SingleSelectEntry.prototype.onPreProcess = function (newValue) {
...
+function ChoiceLabelEntry(question, options) {
+    var self = this;
+    EntrySingleAnswer.call(this, question, options);
+    self.choices = question.choices;
+    self.templateType = 'choice-label';
+
+    self.hideLabel = ko.observable(options.hideLabel);
+
+    self.colStyle = ko.computed(function () {
+        var colWidth = parseInt(12 / self.choices().length) || 1;
```
Comment (sravfeyn) [inline corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js]: What is the significance of this 12 number?

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js [hunk 2]
```diff
@@ -331,6 +331,41 @@ SingleSelectEntry.prototype.onPreProcess = function (newValue) {
     }
 };
 
+/**
+ * Represents the label part of a Combined Multiple Choice question in a Question List
+ */
+function ChoiceLabelEntry(question, options) {
+    var self = this;
```
Comment (sravfeyn) [inline corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js]: `side note` This is the Advaita philosophy in one line, right here! :octocat:

**Review by sravfeyn — APPROVED** [review]
Looks good.

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js (hunk 1 above)
Comment (biyeun) [reply corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js]: we use a 12 column grid system in our layout...

**Pull request merged**